dm/worker/worker.go

```diff
@@ -209,6 +210,12 @@ func (w *Worker) StartSubTask(cfg *config.SubTaskConfig) {
 	// directly put cfg into subTaskHolder
 	// the unique of subtask should be assured by etcd
 	st := NewSubTask(cfg, w.etcdClient)
+	cfg2, err := cfg.DecryptPassword()
+	if err != nil {
+		st.fail(errors.Annotate(err, "start sub task"))
```

Comment:
Do we need to `return` if `DecryptPassword` failed?

If needed, how about putting `DecryptPassword` after `w.subTaskHolder.recordSubTask(st)`, so we can query this error with `query-status` (because the subtask is still existing).

Reply:
yes，I fix it in edca6b4